test/controllers/registration_commands_controller_test.rb

```diff
@@ -40,16 +40,19 @@ class RegistrationCommandsControllerTest < ActionController::TestCase
     end
 
     test 'with params ignored in URL' do
+      features = [FactoryBot.create(:feature, name: 'Registration'), FactoryBot.create(:feature, name: 'Templates')]
+      proxy = FactoryBot.create(:smart_proxy, features: features)
```

Comment:
I'd love if we have additional proxies with only one feature and make sure those won't appear, but it's not crucial :)